core/templates/domain/story/StoryObjectFactory.ts

```diff
@@ -238,6 +246,10 @@ export class StoryObjectFactory {
     );
   }
 
+  /**
+ * TODO(sll) : Remove the interstitial story so that full story can be
+ * created from start.
+ */
```

Comment:
No, TODO comments should be created with an issue number, so create an issue to remove the interstitial story and then do `TODO(#12345):  Remove the interstitial story so that full story…`

Reply:
done!